redbot/core/utils/mod.py

```diff
@@ -124,7 +124,7 @@ def get_audit_reason(author: discord.Member, reason: str = None):
     user = None
     if isinstance(obj, discord.Message):
         user = obj.author
-    elif isinstance(obj, discord.Member):
+    elif isinstance(obj, discord.Member) or isinstance(obj, discord.User):
```

Comment:
Can we just do `isinstance(obj, discord.User)`?

Reply:
no